index.js

```diff
@@ -43,6 +44,16 @@ function defaultKill(input, opts) {
 	return execa(cmd, args);
 }
 
+function processExists(proc, list) {
+	let fn = x => x.pid === proc;
+
+	if (typeof proc === 'string') {
+		fn = x => x.name === proc;
+	}
+
+	list.some(x => fn(x));
+}
```

Comment:
This function never returns anything.

Reply:
We could pause this mr until we know if https://github.com/sindresorhus/process-exists/pull/4 going to be merged or not. this function should be removed if merged :)

Reply:
Added the new feature from sindresorhus/process-exists#4 and removed this